### PR TITLE
UrldataToBack

### PR DESCRIPTION
**Pull request by Anna-rhosyn**
sending shortened and main url to backend

### CODE REVIEW

**Review by JithuPrijith — APPROVED** [review]

File: src/app/components/search-component/search.service.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+console.log(link);
+
+this.http.post('http://localhost:3000/api/post',link);
+
+
+}
+
+
+
+
```
Comment (vajahath) [inline src/app/components/search-component/search.service.ts:39]: extra lines. please fix

File: src/app/components/search-component/search.service.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { Injectable } from "@angular/core";
+import { Router } from "@angular/router";
+import { Link } from "./search.model";
+
+
+
+@Injectable({providedIn:'root'})
+export class SearchService {
+  private links: Link[]=[];
+// private postsUpdated= new Subject< {posts: Post[],postCount:number}>();
```
Comment (vajahath) [inline src/app/components/search-component/search.service.ts:11]: remove this comment

**Review by vajahath — COMMENTED** [review]
Dummy code review

File: src/app/components/search-component/search.service.ts (hunk 1 above)
Comment (vajahath) [reply src/app/components/search-component/search.service.ts:39]: done

**Pull request merged**